RxSwift/Observables/Implementations/TakeLast.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  TakeLast.swift
+//  Rx
+//
+//  Created by Tomi Koskinen on 25/10/15.
+//  Copyright © 2015 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+
+
+class TakeLastSink<ElementType, O: ObserverType where O.E == ElementType> : Sink<O>, ObserverType {
+    typealias Parent = TakeLast<ElementType>
+    typealias E = ElementType
+    
+    private let _parent: Parent
+    
+    private var _elements = [ElementType]()
```

Comment:
I would use `Queue` for `_elements`. `removeFirst` for array has `O(n)` complexity where removing from `Queue` has O(1).